devices/src/virtio/block.rs

```diff
@@ -455,10 +479,12 @@ impl Block {
             );
         }
 
-        let mut avail_features = 1 << VIRTIO_F_VERSION_1;
+        let mut avail_features =
```

Comment:
You also need to implement the actual `flush` if we're now supporting it. A `fsync()` should be enough.

Reply:
Implemented at block.rs:261.